e2e/groups/view-answer.spec.ts

```diff
@@ -129,7 +129,7 @@ test('View answer in logs for current user', async ({ page }) => {
   await expect.soft(page.getByRole('link', { name: 'View answer' })).toBeVisible();
   await page.getByRole('link', { name: 'View answer' }).click();
   await expect.soft(page.locator('h1').getByText('Task with edit tab')).toBeVisible();
-  expect(page.url()).toContain('answerId=6216264238928088073');
+  await expect.soft(page).toHaveURL(new RegExp('answerId=6216264238928088073'));
```

Comment:
I guess you could write
```suggestion
  await expect.soft(page).toHaveURL(/answerId=6216264238928088073/);
```
which would be more readable (applies everywhere)

Reply:
The reason I'm using it is the path with `/`, example:
`new RegExp('/a/home;pa=0')` will be `/\/a\/home;pa=0/`, would be difficult to copy paste